metakube/resource_cluster.go

```diff
@@ -736,28 +740,22 @@ func assignSSHKeysToCluster(projectID, clusterID string, sshkeyIDs []string, k *
 	return nil
 }
 
-func metakubeResourceClusterWaitForReady(ctx context.Context, k *metakubeProviderMeta, timeout time.Duration, projectID, clusterID string) error {
+func metakubeResourceClusterWaitForReady(ctx context.Context, k *metakubeProviderMeta, timeout time.Duration, projectID, clusterID, configuredVersion string) error {
 	return retry.RetryContext(ctx, timeout, func() *retry.RetryError {
 
-		p := project.NewGetClusterHealthV2Params()
+		p := project.NewGetClusterV2Params()
 		p.SetContext(ctx)
 		p.SetProjectID(projectID)
 		p.SetClusterID(clusterID)
 
-		r, err := k.client.Project.GetClusterHealthV2(p, k.auth)
```

Comment:
Without this the function is not checking if the cluster is ready/healthy anymore.

Reply:
addressed